app/javascript/components/widgets/weather/current-weather.jsx

```diff
@@ -30,17 +39,57 @@ const WeatherDescription = ({ baseDescription, weather }) => {
   return baseDescription;
 };
 
+export const AnticipatedPrecipitation = ({ weather }) => {
+  const precip = [];
+  for (let i = 0; i < weather.minutely.length; i += 1) {
+    if (weather.minutely[i].precipitation !== 0) {
+      precip.push([
+        { minute: i },
+        { value: weather.minutely[i].precipitation },
```

Comment:
Could we use a single object with `minute` and `value` keys? That may be a little clearer than an array of arrays of objects.

Reply:
Resolved (: